control_software/config/hera_feng_config.yaml

```diff
@@ -164,18 +165,18 @@ fengines:
         phase_switch_index: [19,20,21,22,23,24]
         ants: [81, 82, 83]
 ## ARP 08/21/20 WR on Node3 not communicating, no phase lock
```

Comment:
Should we add an updated comment here like for Node 9 above?

Reply:
Just did